src/assets/v3/library.11ty.js

```diff
@@ -3,7 +3,7 @@
 const _ = require("lodash");
 const snakeCase = _.snakeCase;
 
-const latestLibraryVersion = "3.109.0";
+const latestLibraryVersion = require("polyfill-library/package.json").version;
```

Comment:
This change means we won't need to update this line when we come to update the version of polyfill-library in the future